Packs/Core/Playbooks/playbook-Cloud_IAM_User_Access_Investigation.yml

```diff
@@ -0,0 +1,900 @@
+      version: -1
+      name: Fetch alert extra data
+      description: Returns information about each alert ID.
+      script: '|||core-get-cloud-original-alerts'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "67"
+    scriptarguments:
+      alert_ids:
+        complex:
+          root: alert
+          accessor: id
+      extend-context:
+        simple: alertData=
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 550,
+          "y": -50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "64":
+    id: "64"
+    taskid: 8eded0e7-c70b-40a6-8515-6a3215682533
+    type: playbook
+    task:
+      id: 8eded0e7-c70b-40a6-8515-6a3215682533
+      version: -1
+      name: Handle False Positive Alerts
+      description: |
+        This playbook handles false positive alerts.
+      playbookName: Handle False Positive Alerts
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "22"
+    scriptarguments:
+      ShouldCloseAutomatically:
+        complex:
+          root: inputs.ShouldCloseAutomatically
+      alertName:
+        complex:
+          root: alert
+          accessor: name
+      sourceIP:
+        complex:
+          root: alert
+          accessor: hostip
+      username:
+        complex:
+          root: alert
+          accessor: username
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 1220,
+          "y": 1250
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "65":
+    id: "65"
+    taskid: 62cfaa81-81fb-4166-8f73-e3e903c5e3c5
+    type: regular
+    task:
+      id: 62cfaa81-81fb-4166-8f73-e3e903c5e3c5
+      version: -1
+      name: Close the alert manually
+      description: Continue the investigation.
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "19"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 320,
+          "y": 1740
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "66":
+    id: "66"
+    taskid: cfb35229-4702-4fa4-8b8b-f9d86778c329
+    type: condition
+    task:
+      id: cfb35229-4702-4fa4-8b8b-f9d86778c329
+      version: -1
+      name: Is it malicious activity?
+      description: Is it a suspicious or Tor IP?
```

Comment:
unclear description

Reply:
Done

Reply:
@OmriItzhak 
Checks wether the alert is malicious based on either one of the following:
1. Source IP address with a malicious DBot score.
2. A Tor IP address.